awx/api/templates/api/instance_health_check.md

```diff
@@ -0,0 +1,33 @@
+{% ifmeth GET %}
```

Comment:
These docs look great. Love that this info is being surfaced. Can take some real splunking otherwise to see how this actually works for awx.